### PR TITLE
Updated readme for V2 release

### PR DESCRIPTION
**Pull request by nosy-b**
A few corrections for the release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 </p>
 
-# iTowns V2
-
 ## What is it?
 
 iTowns is a [Three.js](https://threejs.org/)-based framework written in Javascript/WebGL for visualizing 3D geospatial data.
+It can connect to WMS/WMTS/TMS servers including elevation data and load many different data formats (3dTiles, gpx, KML and much much more).
+
+iTowns has been redesigned from this [early version](https://github.com/iTowns/itowns-legacy)
```
Comment (autra) [inline README.md]: missing `.` at the end.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-<img src="http://www.itowns.fr/images/logo-itowns2XS.png" />
+<img src="https://raw.githubusercontent.com/iTowns/itowns.github.io/master/images/itowns_logo_300x134.png" />
 </p>
 
-# iTowns V2
-
 ## What is it?
 
 iTowns is a [Three.js](https://threejs.org/)-based framework written in Javascript/WebGL for visualizing 3D geospatial data.
+It can connect to WMS/WMTS/TMS servers including elevation data and load many different data formats (3dTiles, gpx, KML and much much more).
```
Comment (autra) [inline README.md]: Can you add a new line between line 8 and 9? I find it more readable

File: README.md [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
+
+iTowns has been redesigned from this [early version](https://github.com/iTowns/itowns-legacy)
 
-It can connect to WMS/WMTS/TMS servers including elevation data and load many different data format (3dTiles, gpx, KML and much much more).
+![iTowns screenshot](https://raw.githubusercontent.com/iTowns/itowns.github.io/master/images/itownsReleaseXS.jpg)
 
-The V2 of iTowns Open Source is a refactoring of [iTowns V1](https://github.com/iTowns/itowns-legacy).
+## API documentation and demo 
+**[API Documentation](http://www.itowns-project.org/itowns2/API_Doc/)**
+**[Demo](http://www.itowns-project.org/itowns2/examples/globe.html)**
```
Comment (autra) [inline README.md]: You have the exact same link in the `examples` section just below. Maybe rename the `Examples` section to `Demo`?

Comment (autra) [reply README.md]: But it's nice to put doc link higher indeed!

**Review by autra — CHANGES_REQUESTED** [review]
Thanks! Just a few remarks, not much though.

**Comment by nosy-b** [comment]
@autra Ok for the mods.
I agree as iTowns is not specifically a globe, I removed the Demo link. Examples will talk for themselves

**Review by autra — APPROVED** [review]

**Comment by autra** [comment]
Thanks!

**Pull request merged**